recipe/meta.yaml

```diff
@@ -21,11 +21,10 @@ source:
     - windows-dll-name.patch
 
 build:
-  number: 5
+  number: 6
   run_exports:
-    # good compatibility in 3.X
-    # https://abi-laboratory.pro/index.php?view=timeline&l=openslide
-    - {{ pin_subpackage('openslide') }}
+    # ABI additions bump minor version, removals bump major version
```

Comment:
For abi compatibility. You only care about breakage. 

Thus an addition would not force recombination of downstream projects. 

The default is ‘X’

And according to your comment. It should stay the pin being used.

You can use your comment to enhance but not remove the previous one.

Reply:
`min_pin` is for downgrade compatibility, no?  This keeps the default `max_pin` of `x`.

The new comment gives upstream's versioning policy.  It doesn't seem useful to append that to a comment speculating about what that policy was ten years ago.